samples/stock/interactive-docs/navigator-styling/demo.js

```diff
@@ -0,0 +1,64 @@
+(async () => {
+
+    // Load the dataset
+    const data = await fetch(
+        'https://demo-live-data.highcharts.com/aapl-ohlc.json'
+    ).then(response => response.json());
+
+    const chart = Highcharts.stockChart('container', {
+        series: [{
+            type: 'ohlc',
+            id: 'aapl-ohlc',
+            name: 'AAPL Stock Price',
+            data: data
+        }]
```

Comment:
Could you set initial range to be eg 1 year? The first thing I clicked was to change the mask from inner to outer and even though I knew the expected result, it was confusing.